undebt/examples/deprecated_inequality_operator.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from pyparsing import Word
+
+from undebt.pattern.python import EXPR
+from undebt.pattern.util import tokens_as_list
+
+
+grammar = EXPR + Word("<>") + EXPR
+
+
+@tokens_as_list(assert_len=2)
```

Comment:
Your pattern actually grabs 3 tokens, so this assertion will fail. See my above comment about suppressing, or change the assertion.